.github/workflows/cypress.yml

```diff
@@ -53,3 +53,22 @@ jobs:
       run: npm run env:start
     - name: Test
       run: npm run cypress:run
+    - name: Update summary
+      if: always()
```

Comment:
I pushed this in 550308bde989dbda4eec7ae7a911de9a7c387c09 to ensure the summary is displayed for both test passes and failures.